packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Writing a unit test
+tags: Mocha, Chai, Enzyme, schemaform-utils,
+---
+
+# Writing a unit test
+
+## Unit testing overview
+
+- `vets-website` uses
+  - [Mocha](https://mochajs.org/) for running unit tests
+  - [Chai](http://chaijs.com/) for test assertions
+  - [Enzyme](https://airbnb.io/enzyme/) for mounting and inspecting React components
+- Unit tests are **collocated with application folder** in a test directory that matches the application directory structure
+```
+- 📂 MyApplication
```

Comment:
Could we include `src` above here and do `my-application` instead of `MyApplication`. I think that helps put it into the context of the vets-website structure better.